components/googleMap/googleMap.js

```diff
@@ -0,0 +1,306 @@
+import React, { Component } from 'react';
+import { PropTypes } from 'prop-types';
```

Comment:
I wasn't aware PropTypes can be imported like this (destructured) from `prop-types` package too.